src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -231,20 +221,16 @@ function FloatingActionButtonAndPopover(props) {
     );
 }
 
-FloatingActionButtonAndPopover.propTypes = propTypes;
-FloatingActionButtonAndPopover.defaultProps = defaultProps;
 FloatingActionButtonAndPopover.displayName = 'FloatingActionButtonAndPopover';
 
-const FloatingActionButtonAndPopoverWithRef = forwardRef((props, ref) => (
+const FloatingActionButtonAndPopoverWithRef = forwardRef<unknown, FloatingActionButtonAndPopoverProps>((props, ref) => (
+    // eslint-disable-next-line react/jsx-props-no-spreading
     <FloatingActionButtonAndPopover
-        // eslint-disable-next-line react/jsx-props-no-spreading
         {...props}
         innerRef={ref}
```

Comment:
remove this props , and use `forwardRef` "normally" which is move `forwardRef` where we are exporting component and in component do `(props, ref) => ...` you can take a look on other components